File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int numsSize) {
    int total=0;
    for(int i=0;i<=numsSize;i++){
        total = total ^ i;
    }
    int given=0;
    for(int i=0;i<numsSize;i++){
        given= given ^ nums[i];
    }
    int final= given ^ total;
    return final;
    
}

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int xorSum=0;
        int n=nums.length;
        for(int i=1;i<=n;i++){
            xorSum^=i;
        }
        for(int num:nums){
            xorSum^=num;
        }
        return xorSum;
    }
}